src/main/java/io/cdap/plugin/gcp/gcs/actions/GCSArgumentSetter.java

```diff
@@ -0,0 +1,244 @@
+/*
+ * Copyright © 2020 AdaptiveScale, Inc.
```

Comment:
@bdmogal do we want to set Copyright © Cask Data, or is AdaptiveScale ok?

Reply:
It this work for google then cask data.